dahu/core/app/scripts/models/actions/move.js

```diff
@@ -14,8 +14,8 @@ define([
         defaults: function() {
             return _.extend({}, ActionModel.prototype.defaults(), {
                 type: 'move',
-                finalAbs: null,
```

Comment:
this change is not without consequence ! What will happen when we will import old dahu presentation ? Did you test an import with an old dahu presentation ? My guess is that  you broke the import... if a change is made here it has to be reflected on the import process: see screencast model.